freshmaker/handlers/koji/rebuild_images_on_parent_image_build.py

```diff
@@ -42,6 +42,17 @@ class RebuildImagesOnParentImageBuild(ContainerBuildHandler):
     name = 'RebuildImagesOnParentImageBuild'
 
     def can_handle(self, event):
+        build_id = event.task_id
+
+        # check db to see whether this build exists in db
+        found_build = db.session.query(ArtifactBuild).filter_by(
+            type=ArtifactType.IMAGE.value,
+            build_id=build_id
+        ).first()
+
+        if not found_build:
+            return False
+
```

Comment:
@qixiang I have a question about this part of code. I am not sure about this solution, because it is duplicated below in method handle. We can not delete this part of code in handle method, because the variable found_builds is used there as well.

Reply:
we can remove this check in `handle` method since the message will always be related after the `can_handle` check.